components/review-sections/IdAndResidencyReviewSection.tsx

```diff
@@ -0,0 +1,83 @@
+import { Heading, HeadingLevels, SummaryList } from "lbh-frontend-react";
+import React from "react";
+import PropTypes from "../../helpers/PropTypes";
+import useReviewSectionRows from "../../helpers/useReviewSectionRows";
+import {
+  idAndResidencyProcessSteps,
+  idAndResidencyResidentSteps,
+} from "../../steps/id-and-residency";
+import Storage from "../../storage/Storage";
+
+interface Tenant {
+  fullName: string;
+  id: string;
+  isPresent: boolean;
+}
+interface IdAndResidencyReviewSectionProps {
+  tenants: Tenant[];
+}
+
+interface TenantSectionProps {
+  tenant: Tenant;
+}
+
+const TenantSection: React.FunctionComponent<TenantSectionProps> = (props) => {
+  const { tenant } = props;
+  const { id, fullName, isPresent } = tenant;
+  return (
+    <>
+      <Heading level={HeadingLevels.H3}>
+        {fullName}
+        {isPresent ? "" : " (not present)"}
+      </Heading>
+      <SummaryList
+        rows={useReviewSectionRows(
```

Comment:
Can we pull the hooks out of being inline to make them easier to find? Feels a bit weird to have it here, and makes conditional rendering difficult to add, with possibly confusing errors. I usually try to keep them at the top of the function, so any conditional logic won't get in the way.

This applies to all of the sections.